README.md

```diff
@@ -34,7 +34,13 @@ tid=923514 'Invalid buffer handle' arg1=2 arg2=0
 The implementation is very simple. The log is a circular ring-buffer of structs:
 
 ```
-    struct {pid_t tid; const char *msg; uint64_t arg1, uint64_t arg2};
+struct {
+    pid_t tid;          // User thread-ID
+    int64_t loc;        // (Optional) Line-of-Code ID
```

Comment:
It's `uint32_t`, no?

Reply:
You are right. It is `uint32_t` for LOC encoding but it will be `int32_t` for LOC-ELF encoding.

I just picked the 2nd typedef as it's more accepting.

Fixed the file.